src/ProjectManager.cpp

```diff
@@ -216,6 +216,10 @@ void InitProjectWindow( ProjectWindow &window )
    // Near as I can tell, this is only a problem under Windows.
    //
 
+   //
+   // Create the Menus and Commands
+   //
+   MenuManager::Get( project ).CreateMenusAndCommands( project );
```

Comment:
This is a simple change,  but I would phrase the commit comment to describe the program’s behavior after the change,  not before.

Reply:
Also add a comment in code, why the ordering of these two calls matters.

Reply:
I have moved function call back to ProjectManager::New, just before the InitProjectWindow.